src/Uno.UI-Skia-only.slnf

```diff
@@ -33,6 +33,7 @@
       "Uno.Sdk\\Uno.Sdk.csproj",
       "Uno.UI.Adapter.Microsoft.Extensions.Logging\\Uno.UI.Adapter.Microsoft.Extensions.Logging.csproj",
       "Uno.UI.Composition\\Uno.UI.Composition.Skia.csproj",
+      "Uno.UI.Composition\\Uno.UI.Composition.Reference.csproj",
```

Comment:
This is going to cascade quite badly I think. We may need to consider re-working the packaging for the Graphics3D package.

Reply:
@Youssef1313  reverted this change. It seems it was not required.